cpp/include/cudf/column/column_device_view.cuh

```diff
@@ -480,26 +497,20 @@ class alignas(16) mutable_column_device_view
 /**---------------------------------------------------------------------------*
  * @brief Returns `string_view` to the string element at the specified index.
  *
- * This function accounts for the offset. Do not call this for a null element.
+ * This function accounts for the offset.
+ * Calling this on a null element will result in undefined behavior.
```

Comment:
>  Calling this on a null element will result in undefined behavior. 

This is a legalese nitpick, but important. This is a deviation from the behavior of `element` for other types. It should be well-defined to call `element` for null elements, what should be undefined is attempting to use the result. 

For example:
```
column_view v{...};

// element 42 is null
string_view element = v.element<string_view>(42); // This is well-defined

if(element.empty()) // Attempting to use the result is undefined
```

A helpful corollary: 
```
void * p{nullptr}; // A pointer to nothing is allowed and well-defined

*p = ...; // Trying to deference (use) the pointer is undefined
```